app/services/whitespace_normalizer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class WhitespaceNormalizer
```

Comment:
Given that there is presently only one place where this class is used, and we never create objects with it, how about we just fold the method `WhitespaceNormalizer.normalize` into `StatementPdf#normalize_whitespace`?

Reply:
I initially broke it out into its own class because it was longer (I was able to shorten it up to just a couple lines) and also it made testing a little clearer. It felt like its own concern. I don't feel too strongly about it.

Reply:
I leave it up to you—I would definitely get rid of it, but I think that’s just a “here’s how I’d do it” thought on me, not a problem 😄